changelog.md

```diff
@@ -4,6 +4,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v6.1.5...HEAD)
+### Changed
+* Bumped streaming client to 13.3.5
+
+### Removed
+* Removed build examples for gulp and browserify and their dependencies
```

Comment:
If you are trying to release this change without any other PRs, you could create a new header. I don't have anything needing to go into a release.

Reply:
I wasn't in a rush to get this out. I was just going to wait until the next release.